server/attributes.h

```diff
@@ -15,6 +15,7 @@ struct attr_methods {
     const char *description;
     /* Set if this attribute contributes to the ATTR change set. */
     bool in_change_set;
+    bool polled_change_set;
```

Comment:
One or two line comment needed to explain.  Looks like this is a special where you format the attribute for polling, but it's not clear to me why this is a thing.